blinkpy/camera.py

```diff
@@ -229,15 +231,29 @@ def extract_config_info(self, config):
         self.name = config.get("name", "unknown")
         self.camera_id = str(config.get("id", "unknown"))
         self.network_id = str(config.get("network_id", "unknown"))
-        self.serial = config.get("serial", None)
+        self.serial = config.get("serial")
         self.motion_enabled = config.get("enabled", "unknown")
-        self.battery_voltage = config.get("battery_voltage", None)
-        self.battery_state = config.get("battery_state", None) or config.get(
-            "battery", None
-        )
-        self.temperature = config.get("temperature", None)
-        self.wifi_strength = config.get("wifi_strength", None)
-        self.product_type = config.get("type", None)
+        self.battery_voltage = config.get("battery_voltage")
+        self.battery_state = config.get("battery_state") or config.get("battery")
+        self.temperature = config.get("temperature")
+        if signals := config.get("signals"):
+            try:  # x / 5 * 100 = x * 20 for percentage
+                self.wifi_strength = signals.get("wifi") * 20
+            except TypeError:
+                self.wifi_strength = None
+            try:
+                self.battery_voltage = signals.get("battery") * 20
```

Comment:
this config.signals.battery field does not appear to be voltage units in the new API, for my cameras, all cameras are currently reporting 3 for this field value.  I suspect it is a 5 point scale like the wifi and lfr fields.  In the previous API, config.battery_voltage appeared to be using units of volts/100 (e.g. 166, 154, etc.).  Is it possible to update the blinkpy field name to be accurate or generic of the units, e.g. "battery_percent" or "battery_strength" instead of "battery_voltage"?  Or maybe you can come up with a transfer function to convert this 5 point scale to a voltage?  It's gonig to be cofusing if you leave the field named voltage but are reporting value that's not a voltage.

Reply:
`battery_level`?